src/domain/app/layout/__tests__/PageLayout.test.tsx

```diff
@@ -68,7 +68,11 @@ it('Pagelayout renders Profile page', async () => {
       request: {
         query: graphql.MyProfileDocument,
       },
-      result: {},
+      result: {
+        data: {
+          myProfile: null,
+        },
+      },
```

Comment:
FYI: https://github.com/City-of-Helsinki/palvelutarjotin-admin/pull/193 also has some changes to my profile tests.